graphql/resolvers/_mutations/publish.js

```diff
@@ -130,11 +161,26 @@ module.exports = async (
   await Promise.all(gitOps)
 
   // cache in redis
-  const doc = await getDocument(
-    { id: commitId, repo: { id: repoId } },
-    { oneway: true },
-    { user, redis }
-  )
+  const repoMeta = await getRepoMeta({ id: repoId })
+  let publishDate = repoMeta.publishDate
+  if (!publishDate) {
+    publishDate = scheduledAt || now
+    await editRepoMeta(null, {
+      repoId,
+      publishDate
+    }, { user, t, pubsub })
+  }
+
+  // transform docMeta
```

Comment:
should this be an util that also is used in `script/pullRedis.js`? Or how does this happen when re-indexing from github?